examples/ref-implementation/keycloak/manifests/keycloak-config.yaml

```diff
@@ -205,7 +205,7 @@ spec:
             name: config-job
       containers:
         - name: kubectl
-          image: docker.io/library/ubuntu:22.04
+          image: ghcr.io/cnoe-io/tools-image:latest
```

Comment:
The github repository which is hosting the code of the container image is empty https://github.com/cnoe-io/tools-image. Is it a mistake ? Do you plan to push the code within the repo ?